examples/tv-app/linux/include/target-navigator/TargetNavigatorManager.cpp

```diff
@@ -52,7 +52,7 @@ std::vector<EmberAfNavigateTargetTargetInfo> TargetNavigatorManager::proxyGetTar
     {
```

Comment:
In all the loops like `for (uint8_t i = 0; i < maximumVectorSize; ++i)`, why is it uint8_t for the index? It would be simpler to just have an int...

Reply:
yes, int is actually more efficient.